packages/expo-dev-launcher/ios/Errors/EXDevLauncherUncaughtExceptionHandler.swift

```diff
@@ -25,11 +25,41 @@ public class EXDevLauncherUncaughtExceptionHandler: NSObject {
     NSSetUncaughtExceptionHandler { exception in
       NSLog("DevLauncher tries to handle uncaught exception: %@", exception)
       NSLog("Stack Trace: %@", exception.callStackSymbols)
+      
+      EXDevLauncherUncaughtExceptionHandler.tryToSendExceptionToBundler(exception: exception)
       EXDevLauncherUncaughtExceptionHandler.defaultHandler?(exception)
     }
   }
 
   static func uninstallHandler() {
     NSSetUncaughtExceptionHandler(defaultHandler)
   }
+  
+  
+  static func tryToSendExceptionToBundler(exception: NSException) {
+    let controller = EXDevLauncherController.sharedInstance()
+    if (controller.isAppRunning()) {
+      guard let url = getLogsUrl(controller) else {
+        return
+      }
+      
+      let logsManager = EXDevLauncherRemoteLogsManager(withUrl: url)
+      logsManager.deferError(message: "Your app just crashed. See the error below.")
+      logsManager.deferError(exception: exception)
+      logsManager.sendSync()
+    }
+  }
+  
+  static func getLogsUrl(_ controller: EXDevLauncherController) -> URL? {
+    let logsUrlFromManifest = controller.appManifest()?.rawManifestJSON()["logsUrl"] as? String
```

Comment:
same comment as on Android -- we should just add a getter to the manifest class instead of accessing the raw JSON (also I think this should be `logUrl`)

Reply:
I will do it in a separate PR